Library/Formula/iso-codes.rb

```diff
@@ -1,9 +1,11 @@
 class IsoCodes < Formula
   desc "ISO language, territory, currency, script codes, and their translations"
-  homepage "http://pkg-isocodes.alioth.debian.org/"
-  url "http://pkg-isocodes.alioth.debian.org/downloads/iso-codes-3.58.tar.xz"
+  homepage "https://pkg-isocodes.alioth.debian.org/"
+  url "https://pkg-isocodes.alioth.debian.org/downloads/iso-codes-3.58.tar.xz"
   sha256 "86af5735dce6e4eff2b983e5d8aa9a3dea1b8db702333ff20be89e45f7f35a72"
 
+  head "git://git.debian.org/git/iso-codes/iso-codes.git"
```

Comment:
Do we need to regenerate the `configure` script here? Upstream ship a `configure` script but not sure how regularly it's updated.